2024-10-23 23:25:11,672 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-23 23:25:11,674 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Configure stats pid to 3109352
2024-10-23 23:25:11,684 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /home/cv-07/.config/wandb/settings
2024-10-23 23:25:11,685 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/settings
2024-10-23 23:25:11,686 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-23 23:25:11,687 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-23 23:25:11,687 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-23 23:25:11,688 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-23 23:25:11,688 INFO    MainThread:3109352 [wandb_init.py:_log_setup():532] Logging user logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/run-20241023_232511-refcdzcy/logs/debug.log
2024-10-23 23:25:11,689 INFO    MainThread:3109352 [wandb_init.py:_log_setup():533] Logging internal logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/run-20241023_232511-refcdzcy/logs/debug-internal.log
2024-10-23 23:25:11,689 INFO    MainThread:3109352 [wandb_init.py:init():617] calling init triggers
2024-10-23 23:25:11,690 INFO    MainThread:3109352 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-23 23:25:11,691 INFO    MainThread:3109352 [wandb_init.py:init():667] starting backend
2024-10-23 23:25:11,692 INFO    MainThread:3109352 [wandb_init.py:init():671] sending inform_init request
2024-10-23 23:25:11,694 INFO    MainThread:3109352 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-23 23:25:11,695 INFO    MainThread:3109352 [wandb_init.py:init():684] backend started and connected
2024-10-23 23:25:11,698 INFO    MainThread:3109352 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-23 23:25:11,703 INFO    MainThread:3109352 [wandb_init.py:init():779] updated telemetry
2024-10-23 23:25:11,747 INFO    MainThread:3109352 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-23 23:26:24,302 WARNING MainThread:3109352 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 815, in init
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
2024-10-23 23:26:25,574 INFO    MainThread:3109352 [wandb_settings.py:_apply_login():1970] Applying login settings: {}
2024-10-23 23:26:25,610 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-23 23:26:25,610 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Configure stats pid to 3109352
2024-10-23 23:26:25,615 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /home/cv-07/.config/wandb/settings
2024-10-23 23:26:25,615 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/settings
2024-10-23 23:26:25,616 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-23 23:26:25,616 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-23 23:26:25,617 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-23 23:26:25,617 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-23 23:26:25,617 INFO    MainThread:3109352 [wandb_init.py:_log_setup():532] Logging user logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/run-20241023_232625-jul3s2le/logs/debug.log
2024-10-23 23:26:25,619 INFO    MainThread:3109352 [wandb_init.py:_log_setup():533] Logging internal logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/run-20241023_232625-jul3s2le/logs/debug-internal.log
2024-10-23 23:26:25,620 INFO    MainThread:3109352 [wandb_init.py:init():617] calling init triggers
2024-10-23 23:26:25,621 INFO    MainThread:3109352 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-23 23:26:25,622 INFO    MainThread:3109352 [wandb_init.py:init():667] starting backend
2024-10-23 23:26:25,627 INFO    MainThread:3109352 [wandb_init.py:init():671] sending inform_init request
2024-10-23 23:26:25,630 INFO    MainThread:3109352 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-23 23:26:25,630 INFO    MainThread:3109352 [wandb_init.py:init():684] backend started and connected
2024-10-23 23:26:25,642 INFO    MainThread:3109352 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-23 23:26:25,644 INFO    MainThread:3109352 [wandb_init.py:init():779] updated telemetry
2024-10-23 23:26:25,674 INFO    MainThread:3109352 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-23 23:27:55,686 ERROR   MainThread:3109352 [wandb_init.py:init():838] encountered error: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-23 23:27:55,691 ERROR   MainThread:3109352 [wandb_init.py:init():1262] error in wandb.init()
Traceback (most recent call last):
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 844, in init
    # Shutdown the backend and get rid of the logger
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2024-10-23 23:28:24,981 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Current SDK version is 0.18.3
2024-10-23 23:28:24,982 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Configure stats pid to 3109352
2024-10-23 23:28:24,986 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /home/cv-07/.config/wandb/settings
2024-10-23 23:28:24,992 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/settings
2024-10-23 23:28:24,997 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-23 23:28:25,002 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-23 23:28:25,007 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-23 23:28:25,012 INFO    MainThread:3109352 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-23 23:28:25,014 INFO    MainThread:3109352 [wandb_init.py:_log_setup():532] Logging user logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/offline-run-20241023_232824-wy0r86v9/logs/debug.log
2024-10-23 23:28:25,019 INFO    MainThread:3109352 [wandb_init.py:_log_setup():533] Logging internal logs to /data/2_data_server/cv-07/link_dl/_02_homeworks/homework_2/wandb/offline-run-20241023_232824-wy0r86v9/logs/debug-internal.log
2024-10-23 23:28:25,024 INFO    MainThread:3109352 [wandb_init.py:init():617] calling init triggers
2024-10-23 23:28:25,030 INFO    MainThread:3109352 [wandb_init.py:init():624] wandb.init called with sweep_config: {}
config: {}
2024-10-23 23:28:25,030 INFO    MainThread:3109352 [wandb_init.py:init():667] starting backend
2024-10-23 23:28:25,032 INFO    MainThread:3109352 [wandb_init.py:init():671] sending inform_init request
2024-10-23 23:28:25,035 INFO    MainThread:3109352 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-23 23:28:25,036 INFO    MainThread:3109352 [wandb_init.py:init():684] backend started and connected
2024-10-23 23:28:25,044 INFO    MainThread:3109352 [wandb_run.py:_label_probe_notebook():1346] probe notebook
2024-10-23 23:28:25,046 INFO    MainThread:3109352 [wandb_init.py:init():779] updated telemetry
2024-10-23 23:28:25,072 INFO    MainThread:3109352 [wandb_init.py:init():812] communicating run to backend with 90.0 second timeout
2024-10-23 23:28:53,384 WARNING MainThread:3109352 [wandb_init.py:init():1256] interrupted
Traceback (most recent call last):
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 1252, in init
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/wandb_init.py", line 815, in init
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/threading.py", line 607, in wait
    signaled = self._cond.wait(timeout)
  File "/data/2_data_server/cv-07/anaconda3/envs/link_dl/lib/python3.10/threading.py", line 324, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
